packages/db/src/models/granules-executions.ts

```diff
@@ -38,6 +38,26 @@ export default class GranulesExecutionsPgModel {
       .merge();
   }
 
+  /**
+   * Get execution_cumulus_id column values from the granule_cumulus_id
+   *
+   * @param {Knex | Knex.Transaction} knexOrTransaction -
+   *  DB client or transaction
+   * @param {number | Array<number>} granuleCumulusIds -
+   * single granule_cumulus_id or array of granule_cumulus_ids
+   * @returns {Promise<Array<number>>} An array of execution_cumulus_ids
+   */
+  async getExecutionCumulusIdsFromGranuleCumulusIds(
```

Comment:
```suggestion
  async searchByGranuleCumulusIds(
```

Pretty subjective, up to you whether you want to take this one

Reply:
Regardless, we need a unit test here as well